### PR TITLE
use-case: Add adaptive retrieval use case

### PR DESCRIPTION
**Pull request by gaocegege**

### CODE REVIEW

File: .vitepress/config.mts [hunk 1]
```diff
@@ -92,26 +103,27 @@ export default defineConfig({
...
-          text: 'FAQs',
+          text: 'Use Cases',
           collapsed: false,
           items: [
-            { text: 'pgvecto.rs vs. pgvector', link: '/faqs/comparison-pgvector' },
-            { text: 'pgvecto.rs vs. specialized vectordb', link: '/faqs/comparison-with-specialized-vectordb' },
-            { text: `Benchmarks`, link: '/faqs/benchmark' },
-          ]
+            { text: 'Hybrid search', link: '/use-case/hybrid-search' },
+            { text: 'Shortening embedding', link: '/use-case/adaptive-retrieval' },
```
Comment (xieydd) [inline .vitepress/config.mts:110]: Do we need to modify the title?

**Comment by VoVAllen** [comment]
Move the use case section upward just after Usage. Other LGTM

File: .vitepress/config.mts (hunk 1 above)
Comment (gaocegege) [reply .vitepress/config.mts:110]: I cannot figure out a good title, I think there are few people know adaptive retrieval. Should we use Matryoshka embedding?

Comment (gaocegege) [reply .vitepress/config.mts]: I changed to adaptive retrieval

**Pull request merged**